ui/src/config.js

```diff
@@ -18,6 +18,7 @@ const defaultConfig = {
   enableStarRating: true,
   defaultTheme: 'Dark',
   enableUserEditing: true,
+  devEnableShare: false,
```

Comment:
These defaults are only used in dev mode, so it is safe to set it to true. Actually, during development, these values are not populated from the config, and this "defaults" are the actual settings, so to see your changes in the UI, you will need it set to true

Reply:
I was hesitant for `true`, thought it would not be safe in master. But I guess I'll bring it back to `false`.
Done.